src/main/sensors/compass.h

```diff
@@ -48,15 +49,16 @@ extern mag_t mag;
 typedef struct compassConfig_s {
     int16_t mag_declination;                // Get your magnetic decliniation from here : http://magnetic-declination.com/
                                             // For example, -6deg 37min, = -637 Japan, format is [sign]dddmm (degreesminutes) default is zero.
-    sensor_align_e mag_align;               // mag alignment
+    flightDynamicsTrims_t magZero;
+    sensorAlignment_t mag_customAlignment;
```

Comment:
This has to go at the end, or the config will be corrupted.

Reply:
the config version was bumped as previously suggested, config version and bumped and re-arranged in a separate commit which had notes in the commit message, as follows:

```
Update compass config.

    * Bump the compass config PG version
    * Re-arrange the values so that all the uint16_t's are aligned.
    * Move magZero so it appears earlier in the structure as it's unlikely
    to be removed/modifed compared to some other values that follow it.
```